code/_expts/slotnick_etal_2001/de_StimCreate.m

```diff
@@ -0,0 +1,346 @@
+function [train,test] = de_StimCreate(stimSet, taskType, opt)
+%Input:
+%  stimSet  : a string specifying which INPUT sets to train autoencoder on
+%                de: (default) 10 "far", 5 "near", 5 "on" images
+%
+%  taskType : a string specifying which OUTPUT task to train on
+%               categorical = compare on (1) to off (0)
+%               coordinate = compare near (1) vs. far (0)
+%
+%  opt      : a vector of options; all listed will be applied
+%
+%OUTPUT: a data file with the following variables:
+%
+%  train.X    : matrix containing 20 vectors, each a unique hierarchical stimulus.
+%  train.T    : target vectors for perceptron (labels, based on task)
+%
+%  test.*     : same as train object, but
+
+  if (~exist('opt','var')),      opt      = {};     end;
+  if (~iscell(opt)),             opt      = {opt};  end;
+  
+  if (~exist('stimSet', 'var')), stimSet  = 'de';   end;
+
+  % Create the input images.
+  train.nInput = [68 50];
+  train.X = zeros(prod(train.nInput), 0);
+  train.XLAB = cell(20, 1);
+  train.TLAB = cell(20, 1);
```

Comment:
Can you mention what `20` means here? Why are there 20 trials? Just refer back to the experiment.